extensions/scarb-snforge-test-collector/src/metadata.rs

```diff
@@ -105,6 +105,16 @@ impl CompilationUnit<'_> {
         }
     }
 
+    /// Retrieve `allow-warnings` flag from the compiler config.
+    pub fn allow_warnings(&self) -> bool {
+        self.unit_metadata
+            .compiler_config
+            .as_object()
+            .and_then(|config| config.get("allow_warnings"))
+            .and_then(|value| value.as_bool())
+            .unwrap_or_default()
```

Comment:
So by default we treat them as errors?

Reply:
Oh, indeed, it is enabled by default in scarb. Thx!

Reply:
Oh, indeed it's enabled by default in scarb! Thx!